docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -2,88 +2,180 @@
 
   You can [monitor the connection state](#receiving-notifications-on-connection-state-changes) to detect when the connection to the remote device is lost.
 
-- Retrieving Bluetooth GATT services     
+- Retrieving Bluetooth GATT services
 
   You can [retrieve information about Bluetooth GATT services](#retrieving-bluetooth-gatt-services) provided by the remote device.
 
   Every GATT service defines characteristics it includes. By knowing the service, you know what features the Bluetooth device exposes.
 
-- Accessing the Bluetooth GATT characteristic value     
+- Accessing the Bluetooth GATT characteristic value
 
   You can [read and write the Bluetooth GATT characteristic value](#accessing-the-bluetooth-gatt-characteristic-value).
 
   Characteristics allows you to monitor and sometimes control remote Bluetooth Low Energy devices. For example, a sensor reading can be exposed by the sensor device as a Bluetooth GATT characteristic.
 
-- Receiving notifications on characteristic value changes     
+- Receiving notifications on characteristic value changes
 
   You can [monitor a characteristic value](#receiving-notifications-on-characteristic-value-changes) to detect any changes, for example, in sensor readings and battery level.
 
-- Accessing the Bluetooth GATT descriptor value     
+- Accessing the Bluetooth GATT descriptor value
 
   You can [read and write the Bluetooth GATT descriptor value](#accessing-the-bluetooth-gatt-descriptor-value).
 
-## Prerequisites
-
-To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/application.html) and [wearable](../../api/latest/device_api/wearable/tizen/application.html) applications) and Bluetooth (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html) and [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html) applications) APIs, the application has to request permission by adding the following privileges to the `config.xml` file:
-
-```
-<tizen:privilege name="http://tizen.org/privilege/application.launch"/>
-<tizen:privilege name="http://tizen.org/privilege/bluetooth"/>
-```
-
 ## Managing the Local Bluetooth Adapter
 
-You can enable or disable the local Bluetooth adapter, and set the device name using the system-provided service through the `ApplicationControl` interface (in [mobile](../../api/latest/device_api/mobile/tizen/application.html#ApplicationControl) and [wearable](../../api/latest/device_api/wearable/tizen/application.html#ApplicationControl) applications).
+You can enable or disable the local Bluetooth adapter, and set the device name using the system-provided service through the `ApplicationControl` interface (in [mobile](../../api/latest/device_api/mobile/tizen/application.html#ApplicationControl), [wearable](../../api/latest/device_api/wearable/tizen/application.html#ApplicationControl) and [tv](../../api/latest/device_api/tv/tizen/application.html#ApplicationControl) applications).
```

Comment:
Global: Please change all headings to verb form and sentence case

Reply:
I have applied your suggestions.